PARTNERS.md

```diff
@@ -69,7 +69,18 @@ sequenceDiagram
 
     Note over Customer:  Upgrade complete
 ```
+
+### kubevirt-velero-plugin mapping to velero and oadp
```

Comment:
would this be better in `Version Mapping` section?

but would remove one `#` either way, its current under `Upgrade workflow` section

Reply:
@mateusoliveira43 you are probably right.. I'll send another pass here in a bit.